src/kubernetes.h

```diff
@@ -147,7 +155,9 @@ class KubernetesUpdater : public PollingMetadataUpdater {
  public:
   KubernetesUpdater(MetadataAgent* server)
       : reader_(server->config()), PollingMetadataUpdater(
-          server, server->config().KubernetesUpdaterIntervalSeconds(),
+          server, 
+          "KubernetesUpdater",
```

Comment:
[Optional] You can put this on the previous line.

Reply:
Done.